server/datastore/mysql/software_installers.go

```diff
@@ -2,14 +2,76 @@ package mysql
 
 import (
 	"context"
-	"database/sql"
+ 	"database/sql"
 
 	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
 	"github.com/fleetdm/fleet/v4/server/fleet"
 	"github.com/google/uuid"
 	"github.com/jmoiron/sqlx"
 )
 
+func (ds *Datastore) ListPendingSoftwareInstallDetails(ctx context.Context, hostID uint) ([]*fleet.SoftwareInstallDetails, error) {
```

Comment:
I know it's not part of the API specs, but I think we should apply a limit (pagination does not really make sense for orbit endpoints), so that if via a script or something a user enqueues a ton of installation requests, we don't harm response times or the database by querying overly large payloads. I believe this API endpoint is used by orbit to get the pending installations to apply? If so, it makes sense to grab a few, process them, send responses (so they are not pending anymore), grab the next batch, etc.

Reply:
This function should only be used as part of the orbit config endpoint, would you suggest limiting the number of returned results?

Reply:
I was, but I see that now it returns only the uuid key, so that should be fine.